File: elixir/unit_test/test/calculator_test.exs
defmodule CalculatorTest do
  use ExUnit.Case
  doctest Calculator

  test "adds two numbers" do
    IO.puts "\nTesting add..."
    assert Calculator.add(2, 3) == 5
  end

  test "subtracts two numbers" do
    IO.puts "Testing subtract..."
    assert Calculator.subtract(5, 3) == 2
  end
end

File: _experimental/_scala/unit_test/demo/src/test/scala/CalculatorTest.scala
import org.scalatest.funsuite.AnyFunSuite

class CalculatorTest extends AnyFunSuite {
  val calculator = new Calculator

  test("Addition of two numbers") {
    assert(calculator.add(2, 3) == 5)
  }

  test("Subtraction of two numbers") {
    assert(calculator.subtract(5, 3) == 2)
  }
}